sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/TimeSeriesInsightsInstancesTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.TimeSeriesInsights.Tests
+{
+    public class TimeSeriesInsightsInstancesTests : E2eTestBase
+    {
+        // TODO: replace hardcoding the Type GUID when the Types resource has been implemented
+        private const string DefaultType = "1be09af9-f089-4d6b-9f0b-48018b5f7393";
```

Comment:
Where does this GUID come from? Does it need to be exactly this?

Reply:
This is the GUID that TSI uses to represent the default type for a Time Series Instance. I am 90% sure about my comment. Unfortunately I cannot confirm this is the case from Dev Tools (F12) because I am having auth issues logging in to any TSI environment. I will check with TSI team and keep this comment open until I confirm.

Reply:
Confirmed with TSI team. This GUID is the default type for any TSI environment.

Reply:
Put that in the comment then.